### ISSUE TITLE
[sort-imports] Single imports that are substrings of other single imports are being flagged

### ISSUE THREAD

**Issue by MeltingMosaic** [buzinas/tslint-eslint-rules#201]
I found a bug where imports that are substrings of other imports are being incorrectly flagged. E.g.:
```typescript
import {Foo} from 'a';
import {Fooz} from 'b'; 
```

I have submitted a PR #200 to fix this.
